provider/postgis/postgis.go

```diff
@@ -128,6 +143,12 @@ func NewTileProvider(config dict.Dicter) (provider.Tiler, error) {
 		return nil, err
 	}
 
+	auto := DefaultAuto
+	auto, err = config.Bool(ConfigKeyAuto, &auto)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Can't we just check to see it there are any layers defined? And if there aren't then assume auto-config.